coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Should this be visible in the changed files?

Reply:
coverage folder is now added to gitignore file.